docs/platform/howto/create-custom-cloud.rst

```diff
@@ -0,0 +1,47 @@
+Create a custom cloud in Aiven |beta|
+=====================================
+
+Learn how to add a custom cloud to your Aiven administrator's account and check out how it works.
+
+About creating a custom cloud
+-----------------------------
+
+You can create a custom cloud in your Aiven administrator's account using `Aiven Console <https://console.aiven.io/>`_.
+
+Limitations
+'''''''''''
+
+
+Prerequisites
+-------------
+
+* Aiven administrator's account
+* Access to `Aiven Console <https://console.aiven.io/>`_
+
+Create a custom cloud
+---------------------
+
+1. Log in to `Aiven Console <https://console.aiven.io/>`_ as an administrator.
+2. From the left sidebar, select **Bring you own account**.
+3. In the **Bring you own cloud** view, select **Create custom cloud**.
+4. In the **Create custom cloud** wizard
+   1. Specify the cloud name, the cloud provider, the region, CIDR, and the deplyment model.
+   2. Select **Generate template**.
+   3. Use the generate template to provide Amazon Resource Name (ARN) in the **ROLE ARN** field.
```

Comment:
Generate an Infrastructure-as-Code (IaC) template using either AWS CloudFormation or Terraform, and run in your AWS account to create the Amazon Resource Name (ARN).  Implement in the "Role ARN" field.